### ISSUE TITLE
www/caddy: Add support for for resolvers option for tls block

### ISSUE THREAD

**Issue by joeroback** [opnsense/plugins#4178]
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:

- [*] I have read the contributing guide lines at https://github.com/opnsense/plugins/blob/master/CONTRIBUTING.md
- [*] I have searched the existing issues, open and closed, and I'm convinced that mine is new.
- [*] When the request is meant for an existing plugin, I've added its name to the title.

**Is your feature request related to a problem? Please describe.**
https://caddyserver.com/docs/caddyfile/directives/tls#resolvers

Add configuration parameter to specify the resolvers in the TLS block of caddyfile

**Describe the solution you'd like**
Seems like this could be global option in the DNS providers tab

**Describe alternatives you've considered**
None, this is just exposing existing caddy options to user

**Additional context**
`tls { ... resolvers 1.1.1.1 1.0.0.1 }`

basically any resolvers specified in the dns providers config, would populate these in each TLS block in caddyfile

thanks

**Comment by joeroback** [comment]
thank you @Monviech
